src/neuroglancer/datasource/deepzoom/base.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc., 2023 Gergely Csucs
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export enum ImageTileEncoding {
+  JPG,
```

Comment:
There should be just one entry for jpeg.

Reply:
Right now both are needed. Inside the XML descriptor a text field comes in specifying the actual extension of files. Later in `backend.js` this extension is reconstructed. `.jpg` and `.jpeg` are both valid extensions.

Reply:
Oh okay.